docs/openapi/paths/surveys.yml

```diff
@@ -0,0 +1,32 @@
+get:
+  summary: Survey list
+  description: Call this API to get the data of a list of survey
```

Comment:
This is missing the `security` property, we need it to add the Bearer Auth for this route 🙏

Reply:
I think it was defined in the global scope following [this guide](https://swagger.io/docs/specification/authentication/#:~:text=Step%202.%20Applying%20security). Please help to check and correct me if I'm wrong 🙏